### PR TITLE
fix(frontend): Onramper widget is scrollable now

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

The Onramper widget was not scrollable for smaller screens, impeding the user to click the Buy button.

https://github.com/user-attachments/assets/ef40b4c9-cae5-4305-9dbb-239bf824c72c

### CODE REVIEW

**Comment by AntonioVentilii-DFINITY** [comment]
[OISY-287]

File: src/frontend/src/lib/components/buy/BuyModal.svelte [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		>{isOnRamperDev ? $i18n.buy.text.buy_dev : $i18n.buy.text.buy}</svelte:fragment
 	>
 
-	<div class="stretch overflow-hidden">
+	<div class="stretch">
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/buy/BuyModal.svelte:14]: `overflow-hidden` was introduced to fix the broken style, sorry for the bug.

that said, if we remove this, then the style is broken again.

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/buy/BuyModal.svelte:14]: I am removing it only here in the BuyModal; I tested the other modals and it doesn't seem to be affecting those

Just to confirm, does that answer the question?

Comment (peterpeterparker) [reply src/frontend/src/lib/components/buy/BuyModal.svelte:14]: That wasn't a question. 😉

See your video, by removing the style you indeed fix the scrolling but you break the style fix provided in #3220 (which unfortunately broke the scroll, as I said, sorry for that).

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/buy/BuyModal.svelte]: Now I see what you mean, I took time (tks for the offline explanation) eheh!

After some tries, have a look at last commit


https://github.com/user-attachments/assets/fb5f52c9-b839-4215-b06c-3175512db31e

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/buy/BuyModal.svelte (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/lib/components/buy/BuyModal.svelte]: COOL

**Pull request merged**